src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  @include fontParams($fontSize, $fontFamily, $fontColor);
+}
+
+.calendar {
+  width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: 90px 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
+    @include cell($cell-size);
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
```

don't forget to create variables transition duration & for all colors

```suggestion
    border: 1px solid $c-black;
    transition: background-color $transitionDuration, transform $transitionDuration;
```